src/routes/+page.svelte

```diff
@@ -99,6 +99,34 @@
   </Row>
 </Section>
 
+<Section id="sponsorships" color="var(--blue)" padding="60px">
+  <Row gap={84} reverse
+    ><figure>
+      <img
+        src={data.info.homepageSponsorships.src}
+        alt={data.info.homepageSponsorships.alt}
+      />
+    </figure>
+    <div id="sponsorship-content">
+      <h2>
+        We
+        <Icon icon="heart" width="1em" height="1em" inline={true} />
+        Our Sponsors
+      </h2>
+      <div class="row-center" />
+      <p>
+        Our drive to write impactful software for social good is made possible
+        by equally socially minded organizations. A sponsorship with Hack4Impact
+        is a lasting relationship that brings us to your commnunity and, in
+        turn, supports our work.
+      </p>
+      <div class="row-center gap">
+        <Button type="primary-white" href="/join/sponsors">For Sponsors</Button>
```

Comment:
Maybe this could say something like "Work With Us". Button labels should be actions.